style/amenity-points.mss

```diff
@@ -165,10 +165,21 @@
     }
   }
 
-  [feature = 'amenity_bus_station'][zoom >= 16] {
-    marker-file: url('symbols/amenity/bus_station.svg');
-    // use colors from SVG to allow for white background
-    marker-clip: false;
+  [feature = 'amenity_bus_station'] {
+    [zoom >= 16] {
+      marker-file: url('symbols/amenity/bus_station.svg');
+      // use colors from SVG to allow for white background
+      marker-clip: false;
+    }
+    [zoom < 16][zoom >=13] {
```

Comment:
Usually we would have this part above, using [feature = 'amenity_bus_station'][zoom >= 13], then [zoom >= 16] below